src/Backend/Modules/FormBuilder/Actions/MassDataAction.php

```diff
@@ -39,7 +42,7 @@ public function execute(): void
             $this->redirect(BackendModel::createURLForAction('Index') . '&error=non-existing');
```

Comment:
I would also do this at the other actions where you had the only actions as default